frontend/src/components/login/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import axios from 'axios'
+
+function Index () {
+  const [emailError, setEmailError] = useState(true)
```

Comment:
Your function here would be LoginForm or Login. The file can be named index because it is the index file in the login folder, but you want your component name to be more descriptive. Let me know if this doesn't make sense.